FreshRSS/FreshRSS
# Improved: share to clipboard improved with animation and icon

Sharing Service `Clipboard`

Before:
User clicks on the `Clipboard` and there is no visual feedback.
![grafik](https://user-images.githubusercontent.com/1645099/231747447-1acc14e4-04b5-4817-948a-232e125b699f.png)


After:
Click on `Clipboard`:
- a background animatoin runs from left to right side (it is not so good visible in the gif)
- a checkmark icon is set, that easeout after 10 seconds
![clipboard](https://user-images.githubusercontent.com/1645099/231748627-9908c802-57d9-4e8b-94cf-9da74d14ca6f.gif)


Notice:
Only the Clipboard is animated, because only this sharing service does not open another service as click feedback.

Changes proposed in this pull request:

- JS and CSS


How to test the feature manually:

1. add `Clipboard` to the sharing service list
2. share an article via click on `Clipboard`
3. see the animation and checkmark

Pull request checklist:
